pkg/crypto/crypto.go

```diff
@@ -15,6 +15,13 @@ const (
 	wmcoMarker = "<wmcoMarker>"
 )
 
+var (
+	tag      = "ENCRYPTED DATA"
+	startTag = "-----BEGIN " + tag + "-----"
+	endTag   = "-----END " + tag + "-----"
+	newLine  = "\n"
```

Comment:
Still not sure why this needs to be a variable

Reply:
Edit: Forgot a `not` in the previous comment

Reply:
this last comment  `Forgot a not in the previous comment` confuses me. do you prefer tag and newLine to be contents or variables ?
I will remove '\n' from variable list.

Reply:
I assume you meant constants and not contents. Yes, they should be constants.